packages/form-builder/src/components/sidebar/tab-panel.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @note This is a fork of the WordPress component.
```

Comment:
@kjohnson to make this more maintainable let's replace this fork of the build file with the actual react component in `@wordpress/components/src/tab-panel/index.js`

Reply:
Oh, did I grab the build file? That explains the strange syntax... 😅

Reply:
@jonwaldstein resolved by https://github.com/impress-org/givewp-next-gen/pull/43/commits/9ff30a146c98d3bf6bc4d80910d15542df3f4130.

That is so much better! Haha. And here I thought the Gutenberg developers were just gluttons for punishment with that syntax.